x-pack/solutions/observability/plugins/observability_ai_assistant_app/scripts/evaluation/README.md

```diff
@@ -12,26 +12,40 @@ This tool is developed for our team working on the Elastic Observability platfor
 
 ## Running evaluations
 
-Run the tool using:
-
-`$ node x-pack/solutions/observability/plugins/observability_solution/observability_ai_assistant_app/scripts/evaluation/index.js`
-
-This will evaluate all existing scenarios, and write the evaluation results to the terminal.
-
 ### Configuration
 
-#### Kibana and Elasticsearch
-
-By default, the tool will look for a Kibana instance running locally (at `http://localhost:5601`, which is the default address for running Kibana in development mode). It will also attempt to read the Kibana config file for the Elasticsearch address & credentials. If you want to override these settings, use `--kibana` and `--es`. Only basic auth is supported, e.g. `--kibana http://username:password@localhost:5601`. If you want to use a specific space, use `--spaceId`
+#### To run the evaluation using a local Elasticsearch and Kibana instance:
 
-#### Connector
+- Run Elasticsearch locally: `yarn es snapshot --license trial`
```

Comment:
`--license trial` is unnecessary no?

Reply:
Without `--license trial`, I can't call the Obs AI Assistant endpoints.

<img width="1004" alt="error-without-license-1" src="https://github.com/user-attachments/assets/d2dbfcfc-053a-4891-9174-2adce5dbac5a" />

<img width="985" alt="Screenshot 2024-12-24 at 8 29 55 AM" src="https://github.com/user-attachments/assets/890cddd5-66d7-4e65-a6fa-8e31ccbb44db" />

Reply:
my bad, I thought `--license trial` was the default, but I simply have it as the default in my command :facepalm:

Reply:
Ah, got it.